src/realm.h

```diff
@@ -3088,6 +3091,16 @@ RLM_API void realm_sync_config_set_recovery_directory_path(realm_sync_config_t*,
 RLM_API void realm_sync_config_set_resync_mode(realm_sync_config_t*,
                                                realm_sync_session_resync_mode_e) RLM_API_NOEXCEPT;
 
+/**
+ * Get latest subscription set for realm  flx sync
+ */
+RLM_API realm_flx_sync_subscription_set_t* get_latest_subscription_set(const realm_t* realm);
```

Comment:
These methods should also be prefixed with `realm_sync` for discoverability and general convention.